src/Akeneo/UserManagement/Bundle/Resources/public/js/form/create-user-button.ts

```diff
@@ -0,0 +1,16 @@
+const CreateButton = require('pim/form/common/index/create-button');
+const FeatureFlags = require('pim/feature-flags');
+
+class CreateUserButton extends CreateButton {
+  public render() {
+    if (FeatureFlags.isEnabled('free_trial') && this.code === 'pim-user-index-create-button') {
```

Comment:
The condition on the extension code `pim-user-index-create-button` looks not mandatory anymore as we use the CreateUserButton where we need.

```suggestion
    if (FeatureFlags.isEnabled('free_trial')) {
```